src/ripple/app/misc/impl/ValidatorList.cpp

```diff
@@ -176,7 +176,10 @@ ValidatorList::load (
         // Config listed keys never expire
         if (it.second)
             it.first->second.expiration = TimeKeeper::time_point::max();
-        it.first->second.list.emplace_back(std::move(*id));
+        if (match.size() == 3)
+            it.first->second.list.emplace_back(*id, match[2]);
+        else
+            it.first->second.list.push_back(*id);
```

Comment:
I don't think this code is doing what you think it is doing.  `match.size()` is a function only of the regex, and not of the search string.  And the regex (`re`) is `const`.  `match.size()` is _always_ 3.